src/lib/parse/forbiddenChars.cpp

```diff
@@ -61,3 +61,36 @@ bool forbiddenChars(const char* s)
 
   return false;
 }
+
+
+
+/* ****************************************************************************
+*
+* forbiddenCharsInUriParam - 
+*/
+bool forbiddenCharsInUriParam(const char* s)
```

Comment:
I have a better idea.

We add a parameter 'char* exceptions' to forbiddenChars, that defaults to the empty string (or NULL).

Where forbiddenCharsInUriParam is called, we call instead forbiddenChars("STRING", ";="), that way telling forbiddenChars to admit ';' and '='.  This easily implemented using **strchr**.

ok?